### ISSUE TITLE
[A11Y] Incorrect or missing text alternative

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1062]
**Finding**
Some images have text alternatives, some don't

**Location**
All images

<img width="177" height="154" alt="" src="https://github.com/user-attachments/assets/6c363b72-1d77-43e4-b24a-9f28a308fb4a" />

<img width="278" height="264" alt="" src="https://github.com/user-attachments/assets/9cf63bc7-095c-4eae-8b25-fb8fce42c4b1" />

<img width="120" height="73" alt="" src="https://github.com/user-attachments/assets/5b25cbb2-84bf-425e-b444-e4c215f9adf1" />

<img width="95" height="85" alt="" src="https://github.com/user-attachments/assets/7cb734bb-db3c-4a8f-a0e1-22a608215c20" />

**Problem and solution**

Not everybody can see images, some people depend on alternatives. These alternatives should be sufficiently valuable that they could replace the image. 

When an image is only decorative (doesn't really add any value or communicate anything), no alternative is needed. Use an empty alt-attribute: `<img alt="">`. Then the image will be ignored.

When an image is inside a link, make sure there is a text alternative. This classifies as a functional image. The text alternative will become the accessible name of the link.
This can be done with the alt-attribute, or a span with a class like "visually-hidden" or "sr-only" (see resources).

When an image communicates something (for example, when it has text), make sure there is a text alternative. Any text in the image should be part of the alternative.

**Resources**
https://www.erikkroes.nl/glossary/#accessible-name
https://www.erikkroes.nl/glossary/#visually-hidden
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
https://www.a11yproject.com/posts/how-to-hide-content/
